java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -223,6 +224,12 @@ public interface StringBaseCommands {
      * String value = client.set("key", "value", options).get();
      * assert value.equals("OK");
      * }</pre>
+     *     <pre>{@code
+     * client.set("key", "value").get();
+     * SetOptions options = SetOptions.builder().conditionalSet(ONLY_IF_EQUAL).comparisonValue("value")).build();
```

Comment:
With customized builder you can do something like
```suggestion
     * SetOptions options = SetOptions.builder().conditionalSetIfEqualTo("value").build();
```

Reply:
Done

Reply:
@Yury-Fridlyand, could you please approve it or provide feedback by Sunday? The exam period starts next week.